### PR TITLE
Hotfix/reviews(srelf)

### PR DESCRIPTION
**Pull request by srelf-ukcloud**

### CODE REVIEW

**Review by dbutts29 — APPROVED** [review]
Approved

**Review by dbutts29 — APPROVED** [review]
Reviewed and approved

**Review by dbutts29 — APPROVED** [review]
Reviewed and approved

**Review by dbutts29 — APPROVED** [review]

**Review by dbutts29 — APPROVED** [review]

File: articles/openstack/ostack-how-use-triliovault.md [hunk 1]
```diff
@@ -48,6 +48,10 @@ Benefits of this solution include:
 
 Before starting to use TrilioVault, it is necessary to know a few commonly used concepts in TrilioVault. Below are some of the important terms in OpenStack and TrilioVault.
 
+## Getting Started
+
+If you are currently using a SSO enabled account to access the OpenStack platform, and you wish to make use of the TrilioVault service, please raise a support ticket and request a Keystone account for Trilio managment.
```
Comment (nebupm) [inline articles/openstack/ostack-how-use-triliovault.md:53]: type on managment.

**Review by nebupm — APPROVED** [review]

**Review by suehighmoor — APPROVED** [review]
LGTM

**Pull request merged**